src/GraphQL/GraphQL.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="System.Reactive.Core" Version="3.1.1" />
     <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
Just noticed this addition.  I'm not seeing `ValueTuple` used anywhere (maybe I'm missing some other usage).  Can this be removed?

Reply:
This allows the use C# tuples `public static (string firstCursor, string lastCursor) GetFirstAndLastCursor<TItem, TCursor>()`

Reply:
The alternative if you want to avoid the dependency is to add a custom struct with these two properties.